test/ufuzz.js

```diff
@@ -573,6 +593,10 @@ function createTypeofExpr() {
         return 'typeof ' + createVarName(MANDATORY, DONT_STORE) + ' != "' + TYPEOF_OUTCOMES[rng(TYPEOF_OUTCOMES.length)] + '"';
       case 4:
         return 'typeof ' + createVarName(MANDATORY, DONT_STORE);
+      case 5:
+      case 6:
+      case 7:
```

Comment:
True. I can only do this if there is a `default:` among these cases. Not sure how common a pattern that would be though.